layers/+tools/shell/packages.el

```diff
@@ -263,10 +263,9 @@
   (evil-define-key 'insert term-raw-map (kbd "C-c C-z") 'term-stop-subjob)
   (evil-define-key 'insert term-raw-map (kbd "<tab>") 'term-send-tab)
 
-  (when (eq dotspacemacs-editing-style 'vim)
-    (evil-define-key 'insert term-raw-map
-      (kbd "C-k") 'term-send-up
-      (kbd "C-j") 'term-send-down))
+  (evil-define-key 'insert term-raw-map
```

Comment:
C-k and C-j need to depend on `dotspacemacs-editing-style` and `hybrid-style-enable-hjkl-bindings`.

Reply:
We need the scroll bindings to be bound somehow, else they are not accessible from insert or hybrid state, which is inconvenient when using a terminal, and especially they are not accessible at all to non-evil users.

Maybe they could be bound to `C-u` and `C-d`, as it is a standard ? Like in `screen`, `less`, `vim` and other ?

Reply:
For scrolling there is C-v and M-v, but there is definitely something missing for non-Evil users. I noticed that <S-prior> (shift+page up), <S-next> (shift+page down), C-c C-p and C-c C-n aren't bound, and I couldn't guess any key binding for scrolling by one line. However, it is an upstream issue - adding bindings there will add them for all users, including Spacemacs users.

Besides, C-j and C-k are really useful in terminals (especially C-k), so overriding them is already questionable.

Reply:
Fair enough, I will put this line back how it was